plugins/palo_alto_cortex_xdr/help.md

```diff
@@ -511,6 +511,8 @@ Example output:
 
 ### Tasks
 
+Plugin Tasks are currently in development! This new plugin capability will collect and deliver events to Insight products for search and use in automation workflows.
+
```

Comment:
@jprice-r7 my only concerns with adding this to individual plugin documentation (granted this is only for one plugin) is that this text will get blown away easily, and it could be forgotten about and live here even after we release tasks. How big of a lift is it to add something quick to the Extension Library?

Reply:
There will be more plugins to have this - there's a backlog of them. I recommend we auto-generate this line in the icon-plugin tooling. The tooling already creates this section in help.md, I imagine we could hardcode this line for any tasks and re-release

Reply:
I should have been more clear in my description; this is already in the Extension Library listing, I opened this PR so the documentation is consistent with what's in the ExtLib. Up to you guys how you want to handle it

Reply:
I think that it is worth merging this PR so that this repo is in sync with what is presented in the Extension Library.